weather_mv/loader_pipeline/metrics.py

```diff
@@ -118,22 +128,48 @@ def process(self, element):
             if not isinstance(time_dict, dict):
                 raise ValueError("time_dict not found.")
 
-            # Adding element processing time.
-            total_time = 0
-            for stage_time in time_dict.values():
-                total_time += stage_time
+            uri = time_dict.pop("uri", _)
 
-            # Converting seconds to milli seconds.
-            self.element_processing_time.update(int(total_time * 1000))
+            time_dict_sorted = dict(
```

Comment:
Lets make use of `OrderedDict`, so that we don't need to sort things out manually :)  .. dict will maintain the `keys` insertion order.

Reply:
Done